mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -361,12 +361,28 @@ func TestAccContainerCluster_withAuthenticatorGroupsConfig(t *testing.T) {
 		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
 		Steps: []resource.TestStep{
 			{
-				Config: testAccContainerCluster_withAuthenticatorGroupsConfig(containerNetName, clusterName, orgDomain),
+				Config: testAccContainerCluster_withAuthenticatorGroupsConfig(containerNetName, clusterName),
```

Comment:
Would it be possible to align the configuration we use w/ authenticator groups config set with a `_basic` config, and use that instead? It's confusing that this is called `_withAuthenticatorGroupsConfig` but doesn't actually set the field.

Reply:
updated